test/http-smoke.js

```diff
@@ -0,0 +1,211 @@
+import _ from 'underscore';
+import chai from 'chai';
+import should from 'should';
+import * as es from 'event-stream';
+import {
+  fixture, bufferJs
+}
+from './fixture';
+import request from 'request';
+import CoreMock from './services/mock-core';
+import MockZKClient from './services/mock-zk';
+import {
+  EventEmitter
+}
+from 'events';
```

Comment:
@rozap sez this comes from `jsformat`. Might as well follow it for now; can decide to change the formatter settings later